components/servo/Cargo.lock

```diff
@@ -1040,6 +1040,7 @@ dependencies = [
  "fnv 1.0.0 (registry+https://github.com/rust-lang/crates.io-index)",
  "geom 0.1.0 (git+https://github.com/servo/rust-geom)",
  "gfx 0.0.1",
+ "gfx_traits 0.0.1",
```

Comment:
Nope, it wasn't.

I noted it but I supposed it was due to some other pr which forgot to update the Cargo.lock files. I'm reverting it :P